### ISSUE TITLE
[Home page] Layout issue in the second slider

### ISSUE THREAD

**Issue by sdgun** [topcoderinc/va-kidney-android#140]
## Steps to Reproduce

1. Login as a valid user
2. Complete the profile data and click Generate Goals
3. Go to Home page
4. Go to the second slider
5. Check the text of the 'Avoid this food' slider

## Actual Result(s)

- Text cut off in the 'Avoid this food' slider

## Expected Result(s)

- Must display the last lines of text clearly

## Frequency

- 5/5

## Environment Details

- **Device(s):** Samsung Galaxy S5
- **Operating System:** Android 6.0.1
- **Size:** 5.1 inches (~69.6% screen-to-body ratio)
- **Resolution:** 1080 x 1920 pixels (~432 ppi pixel density)
- **Browser(s):** Chrome 64.0.3282.137

## Image(s)/Video(s)/Console Log/Crash Log

![issue_20180914_120335](https://user-images.githubusercontent.com/4592712/45564598-91d13b80-b816-11e8-84ac-f0d1192915a9.png)

**Comment by sdgun** [comment]
iOS:

![img_4556](https://user-images.githubusercontent.com/4592712/45581740-1e0c4e80-b869-11e8-85d2-13fdc9443c39.PNG)
![img_4555](https://user-images.githubusercontent.com/4592712/45581741-1e0c4e80-b869-11e8-9fdf-7dcf2195e005.PNG)

**Comment by KevinLeigh** [comment]
@sdgun @talesforce Hey there, please could you let me know the device details such as API and screen size you are using to test with, as it currently is not reproducible on my side.

**Comment by KevinLeigh** [comment]
@sdgun @talesforce Herewith PR to issue -> https://github.com/topcoderinc/va-kidney-android/pull/142, for some reason my original fork is no longer linked so I had to refork and push changes to new fork before creating PR.

**Comment by KevinLeigh** [comment]
@sdgun @talesforce I would suggest running the App with a much smaller device screen and when testing end to end you will see a lot of text cut off issues etc.

**Comment by sdgun** [comment]
Thanks @KevinLeigh you mean highlighted devices in the below list?

**320dp: a typical phone screen (240x320 ldpi, 320x480 mdpi, 480x800 hdpi, etc).**
480dp: a large phone screen ~5" (480x800 mdpi).

**Comment by KevinLeigh** [comment]
@sdgun Yes, I would suggest doing this even if you do not want to support much smaller devices. Just so you get a feel for creating much more responsive layouts that accomodate a wider spectrum of device screens and future screens that may come out, also testing on much smaller screens you will be able to see the potential flaws in the existing layouts that you have created that looks fine on a the device screen it was developed on but not on others and rectify accordingly.

**Comment by sdgun** [comment]
@KevinLeigh Thanks for the info. Appreciate it

**Comment by sdgun** [comment]
Fixed via https://github.com/topcoderinc/va-kidney-android/pull/142
